src/common/task.cc

```diff
@@ -4,6 +4,8 @@
 
 #include "task.h"
 
+#include "ray/gcs/client.h"
```

Comment:
I don't see where this is used within this file.

Reply:
Good call, removed it